cpp/ql/test/library-tests/rangeanalysis/rangeanalysis/RangeAnalysis.ql

```diff
@@ -0,0 +1,12 @@
+import semmle.code.cpp.rangeanalysis.RangeAnalysis
+import semmle.code.cpp.ir.IR
+import semmle.code.cpp.controlflow.IRGuards
+import semmle.code.cpp.ir.ValueNumbering
+
+query predicate instructionBounds(Instruction i, Bound b, int delta, boolean upper, Reason reason) {
+  boundedInstruction(i, b, delta, upper, reason)
+}
+
+query predicate operandBounds(Operand op, Bound b, int delta, boolean upper, Reason reason) {
+  boundedOperand(op, b, delta, upper, reason)
+}
```

Comment:
Why have separate test directories for `RangeAnalysis` and `SimpleRangeAnalysis`? If they share the same C/C++ test files, then it should be easier to compare the two analyses.